website/src/app/page.tsx

```diff
@@ -39,6 +40,17 @@ export default function Page() {
     <>
       <section className="bg-neutral-950 bg-hero pt-28 mb-16">
         <div className="flex flex-col items-center mx-auto md:px-0 px-4 max-w-screen-md">
+          <Banner href="/blog" active>
+            <div className="hover:text-primary-450 transition-all duration-50 ease-in-out invert-0 flex gap-2 mb-5 items-center rounded-full p-1.5 text-neutral-400 bg-[rgba(255,255,255,0.02)] shadow-[inset_0_-8px_11px_0_rgba(255,255,255,0.05)]">
+              <Image
+                src="/images/play-icon.png"
+                width="24"
+                height="24"
+                alt="Play Icon"
+              />
+              <p className="text-sm mr-4">See our latest news</p>
```

Comment:
Shouldn't all of this be in the `Banner` component so we don't need to duplicate it in `children`?